### ISSUE TITLE
When trying to create carousel using loop: true, swiper removes src attributes for <img> tags

### ISSUE THREAD

**Issue by davmaisuradze** [zefoy/ngx-swiper-wrapper#224]
I am trying to create carousel using swiper.js. Here is my configuration:
```
{
    slidesPerView: 1,
    slidesPerColumn: 1,
    initialSlide: this.initialSlide,
    loop: true
}
```

As expected, this will add duplicated slides, near first and last elements. And when I swipe from first to last element, it shows this duplicated blank slide, instead of last one.

As it seems, slides are duplicated incorrectly, because "src" attributes are removed from <img> tag.

Any ideas how to solve this?

![d](https://user-images.githubusercontent.com/49957711/60348955-a3648800-99d1-11e9-9e09-3dfd8df0969e.gif)

**Comment by vkappie** [comment]
Here's stackblitz sandbox: https://stackblitz.com/edit/angular-vppjqn

Issue description: when we have async view generation swiper duplicates only the initial DOM elements. Ideal solution that we can control the duplication elements creation, so we can inject ng components there instead of just DOM. But as I understand swiper doesn't provide such controlling option(e.g. some factory function)

I'm not sure that we can resolve this issue via given API from wrapper or swiper itself.
Let me know if we can resolve this or we have to find some workaround for this

Thanks in advance

**Comment by jroji** [comment]
Hello @vkappie, did you find any workaround to this problem? Im having some troubles with the same problem

**Comment by vkappie** [comment]
> Hello @vkappie, did you find any workaround to this problem? Im having some troubles with the same problem

No, seems there;s no way to overcome it because swiper works only with DOM directly. And since ngx-swiper-wrapper is just a wrapper there's no way to use it properly.
